src/main/java/net/atos/zac/policy/output/TaakRechten.java

```diff
@@ -9,16 +9,23 @@
 import jakarta.json.bind.annotation.JsonbProperty;
 import net.atos.zac.util.DummyInterface;
 
-public record TaakRechten(boolean lezen, boolean wijzigen,
-                          boolean toekennen) implements DummyInterface {
+public record TaakRechten(
+        boolean lezen,
+        boolean wijzigen,
+        boolean toekennen,
+        boolean toevoegenDocument
+) implements DummyInterface {
 
     @JsonbCreator
     public TaakRechten(
             @JsonbProperty("lezen") final boolean lezen,
             @JsonbProperty("wijzigen") final boolean wijzigen,
-            @JsonbProperty("toekennen") final boolean toekennen) {
+            @JsonbProperty("toekennen") final boolean toekennen,
+            @JsonbProperty("toevoegen_document") final boolean toevoegenDocument
+    ) {
```

Comment:
Lenient creation? Use `Boolean` parameters and primitive `boolean` properties?

Reply:
This was in respect to [Postel's Law](https://deviq.com/laws/postels-law), but reconsidering this in respect to internal APIs we should be strict. Nothing to do here 😅